mappings/net/minecraft/block/SmithingTableBlock.mapping

```diff
@@ -1 +1,2 @@
 CLASS net/minecraft/class_3717 net/minecraft/block/SmithingTableBlock
+	FIELD field_22511 SMITHING_TABLE_CONTAINER_NAME Lnet/minecraft/class_2588;
```

Comment:
does it need to specify SMITHING_TABLE? what other container would be defined in SmithingTableBlock?

```suggestion
	FIELD field_22511 CONTAINER_NAME Lnet/minecraft/class_2588;
```